docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -192,15 +146,11 @@ keycloak.init({
 })
 ----
 
-One thing to note is that only an access token is provided and there is no refresh token. This means that once the access token has expired the application
-has to do the redirect to the {project_name} again to obtain a new access token.
+One thing to note is that only an access token is provided and there is no refresh token. This means that once the access token has expired the application has to do the redirect to the {project_name} again to obtain a new access token.
 
 {project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#HybridFlowAuth[Hybrid] flow.
 
-This requires the client to have both the `Standard Flow Enabled` and `Implicit Flow Enabled` flags enabled in the admin console.
-The {project_name} server will then send both the code and tokens to your application.
-The access token can be used immediately while the code can be exchanged for access and refresh tokens.
-Similar to the implicit flow, the hybrid flow is good for performance because the access token is available immediately.
+This requires the client to have both the `Standard Flow Enabled` and `Implicit Flow Enabled` flags enabled in the admin console. The {project_name} server will then send both the code and tokens to your application. The access token can be used immediately while the code can be exchanged for access and refresh tokens. Similar to the implicit flow, the hybrid flow is good for performance because the access token is available immediately.
```

Comment:
```suggestion
This flow requires the client to have both the *Standard Flow*  and *Implicit Flow*  in the Admin Console. The {project_name} server then sends both the code and tokens to your application. The access token can be used immediately while the code can be exchanged for access and refresh tokens. Similar to the implicit flow, the hybrid flow is good for performance because the access token is available immediately.
```